conjur/api.py

```diff
@@ -285,6 +284,56 @@ def update_policy_file(self, policy_id, policy_file):
 
         return self._load_policy_file(policy_id, policy_file, HttpVerb.PATCH)
 
+    def rotate_other_api_key(self, resource, resource_to_rotate):
```

Comment:
I suggest better naming (especially important in signatures)
```suggestion
    def rotate_other_api_key(self, resource_type, resource_name):
```

And this leads me to thing that they can be combined into one type `Resource` that also has the method `id` that does the `':'.join([resource, resource_to_rotate])`.
You could even take it one level further and derive `Host` and `User` from it, which will have the type defined as const for each instance of the class.

Reply:
the api is called by the client whose signature looks like:

```
def rotate_other_api_key(self, resource, resource_to_rotate):
```
Those that use our SDK call `client.rotate_other_api_key('user', 'someuser')`, so we cannot pass in some Resource object.

I can do work inside that function to join `':'.join([resource, resource_to_rotate])` but don't understand what is wrong with the naming `resource_type` / `resource_name`

Reply:
> Those that use our SDK call client.rotate_other_api_key('user', 'someuser'), so we cannot pass in some Resource object.

Don't be so negative and say "cannot". I can suggest some options how to handle this, if you wish.
Anyway, it's up to you.

> don't understand what is wrong with the naming resource_type / resource_name

If I understand correctly, you accepted my rename suggestion, and question the need of an object to wrap them.
The need comes from the fact that these params are related and may repeat multiple times throughout the code.
Moreover, there's more logic specially related to them, so it is clear to me that they deserve a class of their own.

Reply:
as we spoke about, I will update the param as a resource that consists of resource_type and resource_name with id as a function to make that join (: